unstructured_inference/inference/layoutelement.py

```diff
@@ -214,7 +214,9 @@ def reduce(keep: Rectangle, reduce: Rectangle):
             reduce(keep=region_b, reduce=region_a)
 
 
-def table_cells_to_dataframe(cells: dict, nrows: int = 1, ncols: int = 1, header=None) -> DataFrame:
+def table_cells_to_dataframe(
+    cells: List[dict], nrows: int = 1, ncols: int = 1, header=None
```

Comment:
Was the signature for `cells` just wrong before?

Reply:
Yeah, I think so